services/local-notifications/types.go

```diff
@@ -0,0 +1,10 @@
+package localnotifications
+
+const (
+	CategoryTransaction            PushCategory = "transaction"
+	CategoryMessage                PushCategory = "newMessage"
+	CategoryCommunityRequestToJoin              = "communityRequestToJoin"
+
+	TypeTransaction NotificationType = "transaction"
+	TypeMessage     NotificationType = "message"
+)
```

Comment:
I like this, breakout the types of notification. One thing I notice is that `CategoryCommunityRequestToJoin              = "communityRequestToJoin"` doesn't have a type, it is just a string, is this deliberate?

Reply:
nope, missed it, thanks :)